utils/project/path_test.go

```diff
@@ -21,13 +21,27 @@ import (
 )
 
 func TestPackageForPath(t *testing.T) {
-	directory := "./test1"
-	dest := "github.com/caicloud/nirvana/utils/project/test1"
-	packagePath, err := PackageForPath(directory)
-	if err != nil {
-		t.Error(err)
+	tests := []struct {
+		name      string
+		directory string
+		wanted    string
+	}{
+		{
+			name:      "in GOPATH test",
+			directory: "./test1",
```

Comment:
This case indicates that we run this test inside GOPATH, is it always true?

Reply:
Yeah, something wrong here, if you run this test outside GOPATH, you will get wrong results, I can manually edit the value of "goSrcPaths" to simulate that the directory is in the GOPATH